front/app/components/PostShowComponents/Comments/ChildCommentForm.tsx

```diff
@@ -254,11 +264,22 @@ class ChildCommentForm extends PureComponent<Props & InjectedIntlProps, State> {
           focused: false,
         });
       } catch (error) {
+        if (process.env.NODE_ENV === 'development') console.log(error);
+        const apiErrors = get(error, 'json.errors');
+        const profanityApiError = apiErrors.base.find(
+          (apiError) => apiError.error === 'includes_banned_words'
+        );
```

Comment:
I prefer to delegate handling error messages to the error component, in the future, if we want to add more error codes/replace existing, it becomes much easier if all that is required is to change translation strings in `Error/messages.ts`.
The difference is quite big but not enought to change anything now, just a point of view to keep in mind for the future.

Reply:
What would I have to change here then?

Btw, I 💯 agree that this way of dealing with the apiErrors (esp with the missing type info) is not ideal, but I'm being pragmatic with the tight deadline.